main/sw/source/filter/html/swhtml.cxx

```diff
@@ -4413,12 +4398,12 @@ sal_Bool SwHTMLParser::HasCurrentParaFlys( sal_Bool bNoSurroundOnly,
     {
         SwFrmFmt *const pFmt = rFrmFmtTbl[i];
         SwFmtAnchor const*const pAnchor = &pFmt->GetAnchor();
-		// Ein Rahmen wurde gefunden, wenn
-		// - er absatzgebunden ist, und
-		// - im aktuellen Absatz verankert ist, und
-		//   - jeder absatzgebunene Rahmen zaehlt, oder
-		//   - (nur Rahmen oder umlauf zaehlen und ) der Rahmen keinen
-		//     Umlauf besitzt
+		// A frame has been found if
+		// - it is paragraph-bound, and
+		// - it is anchored in the current paragraph, and
+		// - every paragraph-bound frame counts, or
+		// - (only frames or text flow counts) the frame has no
+		//   text flow
```

Comment:
The last two bullet points are indented to the right in the original comment. I think this has a meaning.